src/pages/fr/core-concepts/partial-hydration.md

```diff
@@ -0,0 +1,70 @@
+
+```astro
+---
+// Exemple: Utiliser un composant React dynamique sur une page.
+import MyReactComponent from '../components/MyReactComponent.jsx';
+---
+<!-- Ce composant est maintenant interactif sur votre page !
+     Le reste de votre site reste le même. -->
+<MyReactComponent client:load />
+```
+
+La vaste majorité de votre site est purement en HTML et CSS, avec des **îles d'interactivité** isolées du reste du contenu.
+
+## Hydration Partielle
+
+Il y a de nombreux cas où vous avez besoin d'un composant UI interactif pour s'exécuter dans le navigateur :
+
+- Un slider de présentation d'images
+- Une barre de recherche avec autocomplétion
+- Un bouton pour afficher/masquer une navigation mobile
+- Un bouton "Ajouter au panier"
+
+Dans Astro, c'est votre responsabilité de définir les composants qui doivent être hydratés dans le navigateur. Astro ne va hydrater que ce qui est nécessaire pour le fonctionnement de votre site et laisser le reste de votre site en HTML statique.
+
+**l'hydratation partielle est le secret pour lequel Astro est connu comme "fast-by-default" en terme de performances.**
+
+## Architecture Isolée
+
+**Island architecture (ou Architecture Isolée)** est l'idée d'utiliser l'hydratation partielle pour construire des sites en entier. L'architecture isolée est une alternative à la construction de site sous JavaScript qui doit être envoyé au visiteur de celui-ci.
+
+> L'idée générale d'une architecture "isolée" est simple comme bonjour : rendre des pages HTML sur le serveur, et injecter des emplacements ou des espaces restreint autour de régions dynamiques.
+> <br/> -- [Islands Architecture: Jason Miller (EN)](https://jasonformat.com/islands-architecture/)
+
+Autre que les avantages évidents de ne pas envoyer de JavaScript au navigateur, il y a deux avantages clés à l'architecture isolée :
+
+- **Les composants sont chargés individuellements.** Les composants plus légers (comme une navigation sur téléphone) ne sont pas bloqués par des composants plus lourds de la page.
+- **Les composants sont isolés.** Chaques composants de la page sont isolées, et les performances de la page ne sont pas affectées par les autres.
+
+<IslandsDiagram>
+    <Fragment slot="headerApp">Header "app"</Fragment>
+    <Fragment slot="sidebarApp">Sidebar "app"</Fragment>
+    <Fragment slot="main">
+        Contenu HTML rendu côté serveur comme du texte, des images, etc...
+    </Fragment>
+    <Fragment slot="carouselApp">Carrousel d'images "app"</Fragment>
+    <Fragment slot="advertisement">Publicitée<br/>(rendu côté serveur)</Fragment>
+    <Fragment slot="footer">Footer (HTML rendu côté serveur)</Fragment>
+</IslandsDiagram>
+
+_Source: [Islands Architecture: Jason Miller](https://jasonformat.com/islands-architecture/)_
```

Comment:
Mark the link as "(EN)" in the same way that the previous English-language link is marked.